### DISCUSSION TITLE
Keep server instances loaded in the background

### DISCUSSION THREAD

**Discussion by ThePaulin** [apohl79/audiogridder#1390] [Ideas]
As someone who use Audio Gridder, i think it would be great to be able to keep instances of Audio Gridder loaded in the background to allow faster load times of projects when switching between projects that use the same VST plugins presets as part of the same template. By having a way to manage instances of AG to keep some instances loaded in the background we can speed up load times of projects that use the same AG instances.

**Comment by eakwarren** [comment]
I'd love this feature as well. Basically like the [preserve feature of Vienna Ensemble Pro.](https://www.vsl.co.at/manuals/vep7/preserve-decouple-connect-and-disconnect)
